lib/live_debugger_refactor/services/process_monitor/gen_servers/process_monitor.ex

```diff
@@ -0,0 +1,92 @@
+defmodule LiveDebuggerRefactor.Services.ProcessMonitor.GenServers.ProcessMonitor do
+  @moduledoc """
+  This module is monitoring the status of LiveView processes created by a debugged application.
+  """
+
+  use GenServer
+
+  alias LiveDebuggerRefactor.CommonTypes
+  alias LiveDebuggerRefactor.Bus
+  alias LiveDebuggerRefactor.Services.ProcessMonitor.Actions
+  alias LiveDebuggerRefactor.Services.CallbackTracer.Events.TraceCalled
+  alias LiveDebuggerRefactor.Services.CallbackTracer.Events.TraceReturned
+
+  import LiveDebuggerRefactor.Services.Helpers
+
+  @type state :: %{
+          pid() => MapSet.t(CommonTypes.cid())
+        }
+
+  def start_link(opts \\ []) do
+    GenServer.start_link(__MODULE__, opts, name: __MODULE__)
+  end
+
+  @impl true
+  def init(_opts) do
+    Bus.receive_traces!()
+
+    ok(%{})
```

Comment:
I mean this helper is useful but in such case I'd go the classic way
```suggestion
    {:ok, %{}}
```